mods/tuxemon/db/npc/npc_taba_greeter.json

```diff
@@ -0,0 +1,6 @@
+{
+    "slug": "npc_taba_greeter",
```

Comment:
Except for legacy npcs, all now no longer have npc_ at the beginning. Is there any reason to having it here? npc slugs are only called in npc-specific actions and conditions

Reply:
Right, I'll get that fixed.